images/bridge2/assistant/assistant.go

```diff
@@ -64,6 +66,77 @@ type Agent struct {
 	NewClient         func(name, promptTemplate string) (Client, error)
 }
 
+func (c *Agent) Commands(sess *tracks.Session) commands.Source {
+	return &commands.PrefixedSource{
+		Prefix: "assistant:",
+		Source: commands.NewStaticSource([]commands.Entry{
+			{Name: "add",
+				Def: commands.Def{
+					Description: `add_assistant(name: str, prompt_template: str) -> bool
+			Add an assistant to the session.
+
+			Args:
+				name (str): The assistant's name.
+				prompt_template (str): Template for assistant prompts.
+
+			Returns:
+				success (bool): True if the assistant was added successfully.`,
+					Parameters: commands.FieldDefs{
+						"name": {
+							Name:        "name",
+							Type:        "string",
+							Description: "The assistant's name",
+						},
+						"prompt_template": {
+							Name:        "name",
+							Type:        "string",
+							Description: "Template for assistant prompts",
+						},
+					},
+					Returns: nil,
+				},
+				Run: func(ctx context.Context, args commands.Fields) (commands.Fields, error) {
+					name := args.String("name")
+					name = strings.ToLower(name)
+					prompt := args.String("prompt_template")
+					AddAssistant(sess.SpanNow(), name, prompt)
+					return commands.Fields{
+						"success": true,
+					}, nil
+				},
+			},
+			{Name: "remove",
+				Def: commands.Def{
+					Description: `remove_assistant(name: str) -> bool
```

Comment:
The return types in these descriptions aren't all totally accurate. I know it's still a draft though...